### PR TITLE
72 add movie ids to the database & use in frontend

### PR DESCRIPTION
**Pull request by pswu11**
- [x] add movie and screening modules (express) 
- [x] ingest some movies in the database 
- [x] use own database on Home & Movies pages

Perhaps, a nice-to-have issue would be to improve pagination on Movies where there are more than 5 pages, for now I just slicd it.

### CODE REVIEW

**Comment by ReallyBoard123** [comment]
> Perhaps, a nice-to-have issue would be to improve pagination on Movies where there are more than 5 pages, for now I just slicd it.

For camp9, they got rid of the paginations, and created an endless scroll effect

**Comment by pswu11** [comment]
> > Perhaps, a nice-to-have issue would be to improve pagination on Movies where there are more than 5 pages, for now I just slicd it.
> 
> For camp9, they got rid of the paginations, and created an endless scroll effect

also a good idea.

**Review by ReallyBoard123 — CHANGES_REQUESTED** [review]

File: server/imports/importer.ts [hunk 1]
```diff
@@ -0,0 +1,195 @@
+// ingest movies from TMDB API and generate screenings
+// pnpm node-ts imports/importer.ts
+
+import { Movie, Screening } from '@prisma/client';
+import dotenv from 'dotenv';
```
[the remark this answers is not in the document]
Comment (pswu11) [reply server/imports/importer.ts:5]: @ReallyBoard123 `server/` is not a vite project

**Comment by pswu11** [comment]
@ReallyBoard123 

I made below changes to importer: 
- kept `dotenv` cuz according to Dan vite is more for frontend and for server dotenv is a good fit
- make bindings mutable (`let ...`) instead of `const` - since it's supposed to be mutable 
- increase the request size limit to 5mb for importing larger amount of screenings at once
- add unique constraints to prisma so no duplicated screenings are allowed

**Review by ReallyBoard123 — APPROVED** [review]
to my 👀 this seems good and working!
I wish there was a better senior looking into this for more refined comments!

**Pull request merged**